server.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express'),
+	app = express(),
+	path = require('path'),
+	{ themes, languages, fonts, defaults } = JSON.parse(require('fs').readFileSync(`${__dirname}/config.json`));
```

Comment:
why do we need all these files (fonts, themes, languages) if all of it is inside of config? It seems like each of those files (for ex: `/api/fonts.js`) just import from config and then export config. It could all be done here IMO.

Reply:
yeah i didnt want to change the file structure too much

Reply:
Sure, but it seems like a waste of requiring the file system module and reading in all these files